website/integrations/services/sharepoint-se/index.md

```diff
@@ -0,0 +1,377 @@
+      :::note
+      The minimum is 5 minutes, otherwise SharePoint backend might consider the access code expired.
+      :::
+    - **Access Token validity**: minutes=15
+      :::note
+      The minimum is 15 minutes, otherwise SharePoint backend will consider the access token expired.
+      :::
+    - **Scopes**: select default email, SPopenid and SPprofile
+    - **Subject mode**: Based on the User's hashed ID
+5. Click **Finish**.
+
+### Step 3: Create an application in authentik
+
+From the authentik Admin Dashboard:
+
+1. Open **Applications > Applications** page from the sidebar.
+2. Click **Create** from the application list command bar.
+3. Within the new application form, enter the following values:
+    - **Name**: `auth.applicationName`
+    - **Slug**: `auth.applicationSlug`
+    - **Provider**: `auth.providerName`
+    - (Optional) **Launch URL**: `sp.webAppURL`
+    - (Optional) **Icon**: https://res-1.cdn.office.net/files/fabric-cdn-prod_20221209.001/assets/brand-icons/product/svg/sharepoint_48x1.svg
+4. Click **Create**.
+
+### Step 4: Setup OIDC authentication in SharePoint Server
+
+#### Pre-requisites
+
+##### Update SharePoint farm properties
+
+The following PowerShell script must be updated according to your environment and executed as **Farm Admin account** with **elevated privileges** on a SharePoint Server.
+
+:::caution
+
+-   Update placeholders
+-   Read all script's comments
+
+:::
+
+```PowerShell
+Add-PSSnapin microsoft.sharepoint.powershell
+
+# Setup farm properties to work with OIDC
+$cert = New-SelfSignedCertificate -CertStoreLocation Cert:\LocalMachine\My -Provider 'Microsoft Enhanced RSA and AES Cryptographic Provider' -Subject "CN=SharePoint Cookie Cert"
+$rsaCert = [System.Security.Cryptography.X509Certificates.RSACertificateExtensions]::GetRSAPrivateKey($cert)
+$fileName = $rsaCert.key.UniqueName
+
+#If you have multiple SharePoint servers in the farm, you need to export certificate by Export-PfxCertificate and import certificate to all other SharePoint servers in the farm by Import-PfxCertificate and apply the same permissions as below.
+
+#After certificate is successfully imported to SharePoint Server, we will need to grant access permission to certificate private key.
+
+$path = "$env:ALLUSERSPROFILE\Microsoft\Crypto\RSA\MachineKeys\$fileName"
+$permissions = Get-Acl -Path $path
+
+#Please replace the <web application pool account> with the real application pool account of your web application.
+$access_rule = New-Object System.Security.AccessControl.FileSystemAccessRule("$($env:computername)\WSS_WPG", 'Read', 'None', 'None', 'Allow')
+$permissions.AddAccessRule($access_rule)
+Set-Acl -Path $path -AclObject $permissions
+
+#Then we update farm properties only once.
+$f = Get-SPFarm
+$f.Farm.Properties['SP-NonceCookieCertificateThumbprint']=$cert.Thumbprint
+$f.Farm.Properties['SP-NonceCookieHMACSecretKey']='seed'
+$f.Farm.Update()
+```
+
+##### SharePoint settings in case of SSL Offloading
```

Comment:
```suggestion
##### SharePoint settings in case of SSL offloading
```

Reply:
Sorry I missed this yesterday. We use sentence-case capitalization in our headings.